app/api/ses/webhook/route.ts

```diff
@@ -127,9 +148,14 @@ export async function POST(request: Request) {
       console.error("Failed to retrieve email body from S3:", error);
     }
 
+    if (sender && sender !== "unknown") {
```

Comment:
make the default value for `sender` above `null` instead of `"unknown"`, then raise a 4xx if sender is null